### PR TITLE
Add NOTICE and license files in the executable JAR, with standard plugins v0.10.41

### PR DESCRIPTION
**Pull request by dmikurube**
This pull request adds an appropriate license `NOTICE` file for the executable Embulk distribution binary (`embulk-X.Y.Z.jar`). The Apache License 2.0 requires a software including another Apache-2.0 software ("Derivative Work") to include the `NOTICE` contents of the original software into the Derivative Work.

> If the Work includes a "**NOTICE**" text file as part of its distribution, then any Derivative Works that You distribute must include a readable copy of the attribution notices contained within such NOTICE file, excluding those notices that do not pertain to any part of the Derivative Works, in at least one of the following places: within a NOTICE text file distributed as part of the Derivative Works; within the Source form or documentation, if provided along with the Derivative Works; or, within a display generated by the Derivative Works, if and wherever such third-party notices normally appear. The contents of the NOTICE file are for informational purposes only and do not modify the License. You may add Your own attribution notices within Derivative Works that You distribute, alongside or as an addendum to the NOTICE text from the Work, provided that such additional attribution notices cannot be construed as modifying the License.
>
> https://www.apache.org/licenses/LICENSE-2.0

The executable Embulk binary did not follow this way properly. We wanted to make it right before the next stable v0.10 series.

The embedded standard plugins are ready for the licenses as well, now, with v0.10.41. Then upgrading the standard plugins to v0.10.41 in this pull request, too.

----

At the same time, this pull request adds a new command `embulk license`. It prints out the content of `NOTICE`.

### CODE REVIEW

File: NOTICE-executable [hunk 1]
```diff
@@ -0,0 +1,637 @@
...
+the Apache License, Version 2.0.
+
+* Copyright 2011 The Netty Project
+* The NOTICE.txt file of Netty is embedded in this NOTICE below.
+
+Note that Netty/Common contains modified portions of some other products.
+
+Netty/Common contains the extensions to Java Collections Framework which has
+been derived from the works by JSR-166 EG, Doug Lea, and Jason T. Greene.
+Refer to the license at META-INF/LICENSE-jsr166, instead of license/LICENSE.jsr166y.txt.
```
Comment (hiroyuki-sato) [inline NOTICE-executable:65]: META-INF/LICENSE-jsr166y ?

File: NOTICE-executable [hunk 2]
```diff
@@ -0,0 +1,637 @@
...
+
+* Copyright (c) 2007- Tatu Saloranta, tatu.saloranta@iki.fi
+* The NOTICE file of Jackson JSON processor is embedded in this NOTICE below.
+
+
+
+The executable distribution of this product includes
+Slice, developed by the Airlift project, licensed under
+the Apache License, Version 2.0.
+
```
Comment (hiroyuki-sato) [inline NOTICE-executable:50]: Missing copyright?

File: NOTICE-executable [hunk 3]
```diff
@@ -0,0 +1,637 @@
...
+Netty/Common contains a modified portion of Nitsan Wakart's 'JCTools',
+Java Concurrency Tools for the JVM.
+Refer to the license of JCTools at META-INF/LICENSE-jctools, instead of license/LICENSE.jctools.txt.
+
+
+
+The executable distribution of this product includes
+SnakeYAML, licensed under the Apache License, Version 2.0.
+
+
```
Comment (hiroyuki-sato) [inline NOTICE-executable:79]: Missing copyright?

File: NOTICE-executable [hunk 4]
```diff
@@ -0,0 +1,637 @@
...
+The executable distribution of this product includes
+Codehaus Plexus Utils, developed by
+The Apache Software Foundation (http://www.apache.org/), licensed under
+the Apache License, Version 2.0.
+
+* The NOTICE file of Codehaus Plexus-Utils is embedded in this NOTICE below.
+
+
```
Comment (hiroyuki-sato) [inline NOTICE-executable:162]: Missing copyright?

File: NOTICE-executable [hunk 5]
```diff
@@ -0,0 +1,637 @@
...
+Codehaus Plexus Interpolation, licensed under
+the Apache License, Version 2.0.
+
+
+
+The executable distribution of this product includes
+Codehaus Plexus Component Annotations, licensed under
+the Apache License, Version 2.0.
+
+
```
Comment (hiroyuki-sato) [inline NOTICE-executable:174]: Missing copyright?

File: NOTICE-executable [hunk 6]
```diff
@@ -0,0 +1,637 @@
...
+* The NOTICE file of Codehaus Plexus-Utils is embedded in this NOTICE below.
+
+
+
+The executable distribution of this product includes
+Codehaus Plexus Interpolation, licensed under
+the Apache License, Version 2.0.
+
+
```
Comment (hiroyuki-sato) [inline NOTICE-executable:168]: Missing copyright?

**Review by hiroyuki-sato — CHANGES_REQUESTED** [review]
Could you check my comments?

File: embulk-core/src/main/java/org/embulk/cli/License.java [hunk 7]
```diff
@@ -0,0 +1,85 @@
...
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.PrintStream;
+import java.net.URL;
+import java.util.Enumeration;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * Runs the selfupdate subcommand of Embulk.
```
[the remark this answers is not in the document]
Comment (dmikurube) [reply embulk-core/src/main/java/org/embulk/cli/License.java]: Ah, exactly. Fixed it in: 6cd4259c870eb3435f1e292a4296657345b9a40e

File: NOTICE-executable (hunk 1 above)
Comment (dmikurube) [reply NOTICE-executable]: Fixed it in: f5ecc1b65e0776dc982ee4ed323f18cee8d822f8

File: NOTICE-executable (hunk 2 above)
Comment (dmikurube) [reply NOTICE-executable:50]: I imported copyright notations from the original declarations. I didn't include copyright notations if I was not able to find it in the original source code repositories nor distributions.

**Comment by hiroyuki-sato** [comment]
LGTM👍

**Comment by dmikurube** [comment]
Thanks! -- but the review status still seems "Change requested".

**Review by hiroyuki-sato — APPROVED** [review]
Sorry Approved

**Comment by dmikurube** [comment]
Thanks!

**Pull request merged**